egs2/chime7_task1/diar_asr1/README.md

```diff
@@ -0,0 +1,188 @@
++----+------------+---------------+---------------+----------------------+----------------------+---------------+-----------+----------+-------------+--------------------+--------------------------+-----------------+-----------------+----------------------+--------+-----------------+-------------+--------------+----------+
+####################################################################
+### Macro-Averaged Metrics across all Scenarios (Ranking Metric) ###
+####################################################################
++----+---------------+---------------+---------------+----------------------+----------------------+---------------+-----------+---------+-------------+--------------------+--------------------------+-----------------+-----------------+----------------------+---------+-----------------+-------------+--------------+----------+
+|    | scenario      |   num spk hyp |   num spk ref |   tot utterances hyp |   tot utterances ref |   false alarm |   correct |   total |   confusion |   missed detection |   diarization error rate |   speaker error |   speaker count |   Jaccard error rate |    hits |   substitutions |   deletions |   insertions |      wer |
+|----+---------------+---------------+---------------+----------------------+----------------------+---------------+-----------+---------+-------------+--------------------+--------------------------+-----------------+-----------------+----------------------+---------+-----------------+-------------+--------------+----------|
+|  0 | macro-average |       56.3333 |       48.6667 |                 4784 |                 8373 |       403.417 |   16923.1 | 21946.6 |     1190.88 |            3832.61 |                 0.306686 |         13.7791 |         48.6667 |             0.400192 | 51958.3 |         12486.3 |     14831.3 |      5389.67 | 0.511587 |
++----+---------------+---------------+---------------+----------------------+----------------------+---------------+-----------+---------+-------------+--------------------+--------------------------+-----------------+-----------------+----------------------+---------+-----------------+-------------+--------------+----------+
+```
+We can see that the DER and JER values are quite competitive on CHiME-6 with the state-of-the-art (e.g. JHU CHiME-6 submission) but
+the WER figures are quite poor. <br>
+**Note that here we report DER and JER which are computed against manual annotation for CHiME-6.** <br>
+The DER and JER obtained with respect to the [forced-alignment annotation](https://github.com/chimechallenge/CHiME7_DASR_falign) will be higher. In general this model can be optimized to reach ~46% DER with respect to such annotation on CHiME-6 (thus roughly comparable to Kaldi TS-VAD implementation) but
+we found this configuration to lead to worse WER overall.
+
+This may due to the fact that we use an E2E ASR system which may be more sensitive to segmentation errors compared to
```

Comment:
We could also state the benefit of the simple training recipes thanks to e2e.
As a baseline, we should claim that we consider the balance of simplicity and high performance.

Reply:
I added some lines about it.